spec/models/food_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+RSpec.describe Food, type: :model do
+  create_food = Food.new(name: 'mango', measurment_unit: 'kilograms', price: 10, quantity: 1)
+  subject { create_food }
+
+  before { subject.save }
+
+  it 'name should not be blank' do
+    subject.name = 'mango'
+    expect(subject.name).to eq 'mango'
+  end
+
+  it 'measurment_unit field should not be blank' do
+    subject.measurment_unit = 'kilograms'
+    expect(subject.measurment_unit).to eq 'kilograms'
+  end
+
+  it 'price must be an integer greater than or equal to 0' do
+    subject.price = 3
+    expect(subject.price).to be >= 0
+  end
+end
```

Comment:
- [ ] your tests are failing please make sure to fix that in order for them to pass 
![Screen Shot 2022-11-10 at 19 16 25](https://user-images.githubusercontent.com/61790587/201175320-2fa69e92-dd2b-4f00-a999-16b01ecc9054.png)

Reply:
@youmari I just updated the Gem File and made a fresh install of the RSpec, the food model RSpec test file is now running.